src/black/linegen.py

```diff
@@ -342,9 +347,9 @@ def transform_line(
     ):
         # Only apply basic string preprocessing, since lines shouldn't be split here.
         if mode.experimental_string_processing:
-            transformers = [string_merge, string_paren_strip]
+            transformers = [string_merge, string_paren_strip, hug_power_op]
         else:
-            transformers = []
+            transformers = [hug_power_op]
     elif line.is_def:
         transformers = [left_hand_split]
```

Comment:
Should hug_power_op be in here? Maybe it's easier to just unconditionally add it at the end.

Reply:
Ah yeah I forgot that power ops can exist as parameter default expressions, gah. Good catch!